01week/datatypes.js

```diff
@@ -0,0 +1,80 @@
+'use strict'
+
+// Write a JavaScript program to display the current day and time.
+
+const currentDate = new Date();
+const currentDateAndTime = (currentDate.getMonth() + 1) + '/' + currentDate.getDate() + '/' + 
+currentDate.getFullYear();
+
+console.log(currentDateAndTime);
+
+// Write a JavaScript program to convert a number to a string.
+
+const convertToString = (num1) => num1.toString();
+
+console.log(typeof convertToString(20))
+
+// Write a JavaScript program to convert a string to the number.
+
+const convertToNumber = (stringValue1) => Number(stringValue1);
```

Comment:
this needs to be a function that takes in any string and converts it to a number

Reply:
Hi Renee, just pushed up an updated version.  Thank you.